src/main/java/ur/inf/lab2/pz/servicemanmanagement/domain/validator/FXFormValidator.java

```diff
@@ -3,12 +3,14 @@
 import javafx.scene.text.Text;
 import ur.inf.lab2.pz.servicemanmanagement.utils.StringUtils;
 
+
 public class FXFormValidator {
```

Comment:
Możnaby klasę FXFormValidator zmienić jako abstrakcyjną i dopisać do niej abstrakcyjną metodę validate(), potem klasa ValidUserRegisterForm mogłaby rozszerzać tą klasę i implementować metodę validate(). Dzięki temu w klasie ValidUserRegisterForm będzie można korzystać z metod FXFormValidator i nie będzie trzeba tworzyć jego obiektu. Dzięki temu też później jeśli dojdą inne obiekty formularzy do walidacji (np. NewServiceManForm) to jedynie rozszerzą klasę i zaimplementują validate korzystając z metod nadklasy.

Może lekka przesada, ale taka zabawa polimorfizmem się przydaje więc jak chcesz Damian :D